lib/customer.rb

```diff
@@ -0,0 +1,40 @@
+require 'csv'
+require_relative 'order.rb'
+
+module Grocery
+  class Customer
+    attr_reader :id, :email, :address
+
+    def initialize(id, email, address)
+      @id = id
+      @email = email
+      @address = address
+    end
+
+    def self.all
+      all_customers = []
+      CSV.open("/Users/kimberley/ada/week-three/grocery-store/support/customers.csv", "r").each do |line|
```

Comment:
I suggest avoiding absolute paths like this.  Instead use:  './support/customers.csv'